.gitlab-ci.yml

```diff
@@ -181,35 +148,44 @@ pup5-unit:
   <<: *pup_5
   <<: *unit_tests
 
-pup5.3-unit:
-  <<: *pup_5_3
-  <<: *unit_tests
-
-pup5.5.7-unit:
-  <<: *pup_5_5_7
-  <<: *unit_tests
-
-pup4.10-unit:
-  <<: *pup_4_10
+pup5.5.10-unit:
+  <<: *pup_5_5_10
   <<: *unit_tests
 
 pup6-unit:
   <<: *pup_6
   <<: *unit_tests
-# Acceptance tests
-# ==============================================================================
-#  Note:: Radiusd does not work in fips mode.
-pup5.5.7:
-  <<: *pup_5_5_7
+
+# Acceptance Tests
+#-----------------------------------------------------------------------
+
+pup5.5.10:
+  <<: *pup_5_5_10
   <<: *acceptance_base
   script:
-    - 'bundle exec rake spec_clean'
-    - 'bundle exec rake beaker:suites[default]'
+    - 'bundle exec rake beaker:suites'
 
-pup4.10:
-  <<: *pup_4_10
+pup5.5.10-fips:
```

Comment:
This should not have been added.  See line 201 of the original .gitlab-ci.yml file 'Note:: Radius does not work in fips mode'